tests/wagtail_live/publishers/conftest.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+
+import aioredis
```

Comment:
I needed to add `aioredis==2.0.0` to testenv deps in tox.ini, otherwise tox wouldn't run for me.

Reply:
Ah, this version of aioredis has been released the last week. Prior to that I would do the same but now if I simply do `pip install aioredis` it installs `aioredis 2.0.0`. Do i need to add it to the tox configuration?

Reply:
Well it's not about the version, but when I don't add it I get an error "Module not found aioredis" in tox. So it is required for me to run tox.

Reply:
Ah ok, so I add it in tox?

Reply:
Yes maybe. But it seems odd if you don't have the same problem :-)

Reply:
And the CI passes too so it's odd yes.